pkg/upstream/cluster/cluster.go

```diff
@@ -82,6 +82,32 @@ func NewClusterInfo(clusterConfig v2.Cluster) types.ClusterInfo {
 		info.idleTimeout = clusterConfig.IdleTimeout.Duration
 	}
 
+	// set SlowStart
```

Comment:
It only takes effect when the user configures `slowstart`, and shouldn't  enabled by default.

Reply:
But the value of `info.slowStart.Mode` is never "".
https://github.com/mosn/mosn/blob/ffedd06a6bc259268cfa71ef920a2e2114c38a2f/pkg/upstream/cluster/cluster.go#L85-L91

Reply:
It's my mistake, I will fix it

Reply:
Resolved, PLAT